### DISCUSSION TITLE
山狮帮落网了

### DISCUSSION THREAD

**Discussion by March-777th** [share121/inter-knot#2073] [其他]
<img width="273" alt="Snipaste_2024-08-14_14-11-01" src="https://github.com/user-attachments/assets/ec49531a-d989-4fb3-bf98-c6a74de9a465">

刚从可靠信源得到的消息。媒体还没动静，不过应该很快就会看到相关新闻了，毕竟算得上一件大案。
发帖是为了提醒某些同胞，如果以前替山狮提供过服务...最近还是低调些比较好。

**Comment by BSNHD** [comment]
哥们速度挺快啊<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by Hexiaoling457** [comment]
刚过完剧情

**Comment by hdaihdsu** [comment]
山狮帮，谁？

**Comment by Phaethon-0019** [comment]
治安局动作还挺快

**Comment by KUYUE404** [comment]
本绳匠被揍得好生凄惨QAQ

**Comment by Ye-LinYan** [comment]
帮我一个朋友问一下，治安局大概多久能从嫌犯嘴里撬出合作人信息啊？

**Reply by glhongbo** [reply]
你还是快点跑吧

**Comment by abassado** [comment]
网速最快的一集

**Comment by morrison2077** [comment]
好快.......

**Comment by 00190906** [comment]
假的！别信！我亲眼看到山狮帮的老大带着小弟在打拳！

**Comment by 3022771339** [comment]
在现场

**Comment by Qweulin** [comment]
👍

**Comment by ling333920** [comment]
#<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3d389fd2885b008e46d843c552c02bf3_5073438697399152551.png" alt="undefined" width="75" >

**Comment by 13671406204** [comment]
3a哥（指3拳秒）
